server/tests/controllers/users.js

```diff
@@ -145,4 +148,170 @@ describe('Users Controllers', () => {
         });
     });
   });
+  describe('recover password', () => {
+    it('should take a payload and send a mail to the email provided', (done) => {
+      chai.request(app)
+        .post('/api/users/password/recover')
+        .send({
+          user: {
+            email: user.email,
+          },
+          links: {
+            reset: 'https://localhost:5500',
```

Comment:
You have another reset link to 'https://localhost:5500', I would like to know why you are having different reset link addresses